Owin.Security.Providers/LinkedIn/Provider/LinkedInAuthenticatedContext.cs

```diff
@@ -69,12 +70,17 @@ public LinkedInAuthenticatedContext(IOwinContext context, JObject user, string a
         /// </summary>
         public string Name { get; private set; }
 
-        public string Link { get; private set; }
+        /// <summary>
+        /// Get the user's first name
+        /// </summary>
+        public string GivenName { get; private set; }
 
         /// <summary>
-        /// Gets the LinkedIn username
+        /// Get the user's last name
         /// </summary>
-        public string UserName { get; private set; }
```

Comment:
The UserName property is not being used anymore. It was being set as the `name` claim type. The name claim is actually set to the LI `formatedName` now, which is the correct approach to take. If someone was using the name claim to get this arbitrary username, which was just the LI `formattedName` without spaces, then they will need to strip spaces from the name claim on their own now (the way if should have been done in the first place). That will be the only "backwards compatibility" that gets broken. 

It was incredibly confusing to me when using this package as to why the Name claim was incorrect from what LI was returning. This change allows future implementations of the library to be accurate with what comes from LI, so users of the package don't have to go digging through source code like I did.